### ISSUE TITLE
Feature request - bookmark

### ISSUE THREAD

**Issue by gorgoserio** [sendyhalim/Yomu#2]
Hi, first of all nice app, it would be very nice if you could add the feature of bookmarking the manga with the latest chap read. 

Thx for your work, A. G.

**Comment by sendyhalim** [comment]
@gordonwes sure that's a good idea 👍 I'll work on it as soon as I have spare time. So let's discuss the detail
- I have manga A 
- I click manga A
- I read chapter 80
- There should be a bookmark chapter button 
- I click it (bookmark it)
- The next time I open the manga, the manga reader will automatically mark the bookmarked chapter (maybe with a different background?)

:beers:

**Comment by gorgoserio** [comment]
Yeah, the best would be if there is a 'bookmark this' button, but for the entire manga, this way it store the read notification for the read chap (only when scroll up to the last 2 page of the chap, so that if you open for mistake a chap it doesn't store) of the bookmarked manga. 
Considering that most of the manga out there are already very long, if you aren't start a reading from chap 1 it should ask 'want mark as read all the previous chap?'.
For the style on the chap i would go with different background, or simply with a minimal icon on the right, or both :). There should also be a unread badge count (at least a simple notification) on the manga itself.
For the rest i have no comments to make, the graphic is very clean, i like it, the only thing is maybe adding a advance search feature, with filter for genres, but for me is an extra, i don't really care.
Keep the good work, cheers!

A.G
